org.eclipse.xtext/src/org/eclipse/xtext/validation/DiagnosticConverterImpl.java

```diff
@@ -239,21 +244,28 @@ protected IssueLocation getLocationData(EObject obj, EStructuralFeature structur
 					containingFeature.isMany() ? ((EList<?>) container.eGet(containingFeature)).indexOf(obj)
 							: ValidationMessageAcceptor.INSIGNIFICANT_INDEX);
 		}
-		IssueLocation result = new IssueLocation();
-		result.lineNumber = 1;
-		result.column = 1;
-		result.offset = 0;
-		result.length = 0;
-		return result;
+		return new IssueLocation();
```

Comment:
We used to init line and column to one. The new default is zero. We should stick to the old default for this case

Reply:
It looks to me as if the contract*) was violated by the old implementation.

*) contract being
> Please also state that a value '0' indicates 'no line information'

Reply:
At this particular point, we wanted to create an artificial location for a diagnostic, if we were not able to figure the "right" location. E.g. put it on the first line before the first char (offset zero, length zero, line 1, column 1)

Reply:
In that case, I can just set some "valid" values for lines and columns.

Reply:
:+1: